### PR TITLE
Move misplaced files (InvalidDepthOneFile)

### PR DESCRIPTION
**Pull request by dorschw**
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10838

### CODE REVIEW

File: Packs/DelineaDSV/Integrations/README.md [hunk 1]
```diff
@@ -1,67 +0,0 @@
-Manage credentials for applications, databases, CI/CD tools, and services without causing friction in the development process.
```
Comment (yuvalbenshalom) [inline Packs/DelineaDSV/Integrations/README.md:1]: shouldn't this be moved to pack?

File: Packs/GLIMPS_Detect/Integrations/README.md [hunk 2]
```diff
@@ -1,5 +0,0 @@
-# GLIMPS Detect
-
-This Content Pack is used to create an XSOAR integration for GDetect. Thanks to this integration, submit files to GLIMPS Malware
```
Comment (yuvalbenshalom) [inline Packs/GLIMPS_Detect/Integrations/README.md:3]: shouldn't this be moved to pack?

**Review by yuvalbenshalom — COMMENTED** [review]
looks good. see my comments.

File: Packs/DelineaDSV/Integrations/README.md (hunk 1 above)
Comment (dorschw) [reply Packs/DelineaDSV/Integrations/README.md:1]: there is one in the pack level

File: Packs/GLIMPS_Detect/Integrations/README.md (hunk 2 above)
Comment (dorschw) [reply Packs/GLIMPS_Detect/Integrations/README.md:3]: there is one in the pack level

**Review by yuvalbenshalom — APPROVED** [review]

**Comment by dorschw** [comment]
@yuvalbenshalom please force merge, files are considered as deleted but are either redundant or moved around.

**Pull request merged**